packages/labs/testing/src/lib/litSsrPlugin.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {Worker} from 'worker_threads';
+import * as pathlib from 'path';
+import {litSsrPluginCommand} from './constants.js';
+
+import type {TemplateResult} from 'lit';
+import type {TestRunnerPlugin} from '@web/test-runner';
+
+export interface Payload {
+  template: TemplateResult;
+  modules: string[];
+}
+
+export function litSsrPlugin(): TestRunnerPlugin<Payload> {
+  return {
+    name: 'lit-ssr-plugin',
+    async executeCommand({command, payload}) {
+      if (command === litSsrPluginCommand) {
```

Comment:
Suggest early return here

```suggestion
      if (command !== litSsrPluginCommand) {
        return undefined;
      }
```